src/utils/__tests__/conversion.ts

```diff
@@ -1193,35 +1231,47 @@ describe('permissionsToMeshPermissions and meshPermissionsToPermissions', () =>
     const ticker = 'someTicker';
     const did = 'someDid';
     let fakeResult: Permissions = {
-      tokens: [entityMockUtils.getSecurityTokenInstance({ ticker })],
-      transactions: [
-        TxTags.identity.AddClaim,
-        TxTags.confidential.AddRangeProof,
-        TxTags.confidential.AddVerifyRangeProof,
-      ],
-      transactionGroups: [],
-      portfolios: [entityMockUtils.getDefaultPortfolioInstance({ did })],
+      tokens: {
+        values: [entityMockUtils.getSecurityTokenInstance({ ticker })],
+        type: PermissionType.Include,
+      },
+      transactions: {
+        type: PermissionType.Include,
+        values: [ModuleName.Identity, ModuleName.Confidential],
```

Comment:
This would suggest permissions over the entire Identity module, which is not what the `permissions` variable suggeests.

Also, having the Identity module as a whole and then also having `ClaimsManagement` as a group is redundant

Reply:
This too is being fixed in another PR, so ignore